src/Components/Publishing/Display/DisplayPanel.tsx

```diff
@@ -12,12 +12,17 @@ interface DisplayPanelProps extends React.HTMLProps<HTMLDivElement> {
 }
 
 interface DivUrlProps extends React.HTMLProps<HTMLDivElement> {
-  imageUrl: string
-  hoverImageUrl: string
+  imageUrl?: string
+  hoverImageUrl?: string
+  coverUrl?: string
+  onMouseEnter: any
+  onMouseLeave: any
```

Comment:
How does one play the video on mobile?

Reply:
Oh good point, cc @owendodd

Reply:
There is also no mobile alternative for the image layout, assuming that is ok

Reply:
A quick solution could be adding a play icon
<img width="363" alt="screen shot 2017-10-31 at 9 06 16 pm" src="https://user-images.githubusercontent.com/1497424/32255585-6534aef4-be7f-11e7-8fe1-b9c54dcd7d92.png">

Reply:
Yeah good call. This would work well!